src/pages/Dashboard/Activities/AddForm/index.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import AddForm from '.';
+import * as constant from './constant';
+
+test('logic test', async () => {
+  const mockOnSubmit = jest.fn();
+
+  render(<AddForm onSubmit={mockOnSubmit} />);
+
+  expect(screen.getByText(constant.FORM_HEADLINE)).toBeInTheDocument();
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_TITLE_PLACEHOLDER), 'Bowling');
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_ADDRESS_PLACEHOLDER), '123 address, Montreal, QC, Canada, EXM PLE');
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_STARTTIME_PLACEHOLDER), '01-09-2022');
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_ENDTIME_PLACEHOLDER), '01-09-2023');
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_DESCRIPTION_PLACEHOLDER), 'Bowl until you cant bowl no more');
+  userEvent.clear(await screen.findByPlaceholderText(constant.INPUT_COST_PER_INDV_PLACEHOLDER));
+  userEvent.type(await screen.findByPlaceholderText(constant.INPUT_COST_PER_INDV_PLACEHOLDER), '9.99');
```

Comment:
Because for some reason there is an input of 0 as a placeholder for any number type so if you just add for example '9.99' it becomes '09.99' which is not the same. This error was causing the test to fail so I fixed it by first clearing the input field.

Reply:
ah ok makes sense